### DISCUSSION TITLE
Grid Layout sorting/order?

### DISCUSSION THREAD

**Discussion by jpmoo** [HEmile/juggl#68] [Q&A]
Hello!

I was looking at Juggl to possibly help me create a daily journal/mood tracker. The idea here would be to use the grid layout and style groups to give each note a different color based on the presence of certain tags connected to mood. If I pointed Juggl at a daily notes folder, and each note had just one of these mood tags in it, I'd get a pretty decent map of my moods over time.

What order do the notes appear in the grid layout? Date order? Ordered by title? Some other order?

**Comment by jpmoo** [comment]
Did some experimenting here and discovered that it's ordering them by filename. Using style groups and grid layout, I can get a really nice mood tracker. 

I added a section to my daily note template with a series of emojis. I put a hashtag in front of the one for that day, and point the style groups at the hashtag + emoji. It's working like a charm!

**Reply by HEmile** [reply]
Oh wow, that sure is original and cool! Glad to see it works out :)
